src/admin/client.ts

```diff
@@ -74,7 +76,7 @@ class DisplayOrder {
 
     public submit = () => {
         var msg = new Models.OrderRequestFromUI(this.side, this.price, this.quantity, this.timeInForce, this.orderType);
-        this._log.info("submitting order", msg);
```

Comment:
thanks! marked as TODO soon (will try option A)

about the vmdk of docker comment, i found the hard disk of my virtual boxes completely full after a few days of run.